Upstream: Should be, and they should fix it
Reason: Fixes a few issues

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed6e603..5f899c1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,14 +34,8 @@ set_if(GNU         ${CMAKE_SYSTEM_NAME} MATCHES "GNU")
 set_if(POSIX       LINUX OR DARWIN OR BSD OR (CYGWIN AND CYGWIN_USE_POSIX))
 set_if(SYMLINKABLE LINUX OR DARWIN OR BSD OR CYGWIN OR GNU)
 
-# Not sure what to do in case of compiling by MSVC.
-# This will make installdir in C:\Program Files\SRT then
-# inside "bin" and "lib64" directories. At least this maintains
-# the current status. Shall this be not desired, override values
-# of CMAKE_INSTALL_BINDIR, CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_INCLUDEDIR.
-if (NOT DEFINED CMAKE_INSTALL_LIBDIR)
-	include(GNUInstallDirs)
-endif()
+
+include(GNUInstallDirs)
 
 set (SRT_VERSION 1.4.2)
 set_version_variables(SRT_VERSION ${SRT_VERSION})
@@ -59,14 +53,6 @@ endif()
 # Set CMAKE_BUILD_TYPE properly, now that you know
 # that ENABLE_DEBUG is set as it should.
 
-if (ENABLE_DEBUG EQUAL 2)
-	set (CMAKE_BUILD_TYPE "RelWithDebInfo")
-elseif (ENABLE_DEBUG) # 1, ON, YES, TRUE, Y, or any other non-zero number
-	set (CMAKE_BUILD_TYPE "Debug")
-else()
-	set (CMAKE_BUILD_TYPE "Release")
-endif()
-
 message(STATUS "BUILD TYPE: ${CMAKE_BUILD_TYPE}")
 
 getVarsWith(ENFORCE_ enforcers)
